dkron/job.go

```diff
@@ -303,10 +305,44 @@ func (j *Job) GetTimeLocation() *time.Location {
 	return loc
 }
 
+// scheduleHash replaces H in the cron spec by a value derived from job Name
+// such as "0 0 H * * *"
+func (j *Job) scheduleHash() string {
+	spec := j.Schedule
+	if strings.Contains(spec, "H") && strings.Count(strings.TrimSpace(spec), " ") == 5 {
+		h := 0
+		for _, c := range j.Name {
+			h += int(c)
+		}
+		parts := strings.Split(spec, " ")
+		for index, part := range parts {
+			if strings.Contains(part, "H") {
+				// mods taken in accordance with https://dkron.io/docs/usage/cron-spec/#cron-expression-format
+				ph := h
+				switch index {
+				case 2:
+					ph %= 24
+				case 3:
+					ph = (ph % 31) + 1
```

Comment:
Not all months are 31 days long.

Reply:
While for hours I didn't understand your point, I do in this one 😄 
Updated max to `28` 🙇 
Curious cron library would handle choosing day 31 for a month of 30, as anyone can specify that in a spec, will it skip a month?